geocity/apps/accounts/auth_backends.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+
+
+class EmailAuthenticationBackend(ModelBackend):
+    def authenticate(self, request, email, password, **kwargs):
+        User = get_user_model()
+
+        if not email or not password:
+            return None
+
+        try:
+            user = User.objects.get(email=email)
```

Comment:
This fails when database contains multiple users with same email.

Should we transform it to `User.objects.filter(email=email).first()` to avoid errors, or make some kind of a data migration to set duplicate emails to NULL and send a warning email?

Anyway, the email field of the User model cannot be changed to `unique=True`, since a custom User model was not used.

I'm waiting on a policy choice :)

Reply:
I did the `first()` then :)